### DISCUSSION TITLE
What’s the database for?

### DISCUSSION THREAD

**Discussion by WoisWoi** [NginxProxyManager/nginx-proxy-manager#1281] [Q&A]
Hi :)
What’s the database for ? 

Is it only to store the connection ID ?

I ask because the database conflicts with another mysql database that I use for wordpress making my site unusable.

**Comment by TechTucson** [comment]
If you change the database name to something other than "npm" would that work?, or could it be the volume you are using matches up to the WordPress container?

**Reply by WoisWoi** [reply]
It's not the same volume.
![image](https://user-images.githubusercontent.com/75491231/128531913-e521591a-5406-4372-b815-abf55d449058.png)
![image](https://user-images.githubusercontent.com/75491231/128531982-9236a6a0-15b3-4708-b411-c8e01ffca7cf.png)

![image](https://user-images.githubusercontent.com/75491231/128532046-1f880bc8-7a02-4928-8e5f-97e6157dcfc3.png)
![image](https://user-images.githubusercontent.com/75491231/128532110-d3c9a528-4918-48c6-882f-8eb4d5778d65.png)
